app/views/api/v1/users/update.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.user current_user, partial: 'info', as: :user
```

Comment:
As this is the same as in the `show` view, you can remove this view and call `render :show` at the end of the `update` action in the controller